CONTRIBUTORS.md

```diff
@@ -33,3 +33,4 @@ Wails is what it is because of the time and effort given by these great people.
   * [artem](https://github.com/Unix4ever)
   * [Tim Kipp](https://github.com/timkippdev)
   * [Dmitry Gomzyakov](https://github.com/kyoto44)
+  * [Ilgıt Yıldırım](https://github.com/ilgityildirim)
```

Comment:
Sneaky :)

Reply:
I prefer this. Saves me from trying to find out the information 😃